analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -211,7 +210,8 @@ class GoMod(
     )
 
     /**
-     * Return the list of all modules contained in the dependency tree with resolved versions.
+     * Return the list of all modules contained in the dependency tree with resolved versions, with the 'replace'
```

Comment:
I deliberately proposed to use "and" because the "with ... with" word duplication reads odd, IMO, but I won't insist on a change.